### DISCUSSION TITLE
Personal Access Token to deploy to Microsoft vscode marketplace expired

### DISCUSSION THREAD

**Discussion by bhufmann** [eclipse-cdt-cloud/cdt-cloud#45] [General]
I just noticed that the Personal Access Token PAT to deploy the `vscode-trace-extension` to Microsoft marketplace expired and needs to be renewed. It uses the `eclipse-cdt` publisher for that.

How can I renew the PAT for `eclipse-cdt` cloud? Please let me know

Thanks

**Comment by planger** [comment]
I think you'll need to request a new PAT to be configured for the Github organization via the [Eclipse Helpdesk](https://gitlab.eclipse.org/eclipsefdn/helpdesk).

**Comment by bhufmann** [comment]
I've opened a support request: 
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4908

**Reply by bhufmann** [reply]
The PAT was updated and I'm able to publish a new version of the vscode-trace-extension. However, looking on the market place, the new version is not published, only released. It states Published `2024-03-16, 13-37:46`, but `Last Released: 2024-09-20, 13:04:59` which matches the github workflow execution. I did some internet search, I noticed that others had similar issues and they stated that they got an email the the validation failed. I guess, the email address used is the one associated with the publisher. Does anybody know who receives that email for the `eclipse-cdt` publisher?

**Reply by planger** [reply]
I don't receive those emails unfortunately.

**Reply by bhufmann** [reply]
@marcdumais-work confirmed with @thegecko and publisher owners will receive such emails. The received logs suggested to try it again. So, I did another release and it worked this time. Thanks for the support.
